website/docs/pipelines/promoting-applications.mdx

```diff
@@ -140,97 +141,60 @@ is upgraded in one of the pipeline's environments, a PR is created that updates
 The dynamic nature of GitOps deployments requires you to assist Weave GitOps a little with information on which repository hosts the manifest files,
 how to authenticate with the repository and the Git provider API, and which file hosts the version definition for each environment.
 
-:::caution
+### Security
```

Comment:
Extract the general recommendations to a general-purpose page and reference it here. it is required for other cases.

Reply:
We will refactor after merged so we dont increase scope from the original one.